ERROR - 2023-07-19 05:48:20 --> Error connecting to the database: ErrorException: mysqli::real_connect(): php_network_getaddresses: getaddrinfo for sql108.infinityfree.com failed: No such host is known.  in E:\XAMPP\htdocs\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'mysqli::real_co...', 'E:\\XAMPP\\htdocs...', 173)
#1 E:\XAMPP\htdocs\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('sql108.infinity...', 'if0_34583510', Object(SensitiveParameterValue), 'if0_34583510_ti...', 3306, '', 0)
#2 E:\XAMPP\htdocs\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#3 E:\XAMPP\htdocs\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#4 E:\XAMPP\htdocs\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#5 E:\XAMPP\htdocs\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#6 E:\XAMPP\htdocs\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#7 E:\XAMPP\htdocs\app\Controllers\Main.php(34): CodeIgniter\BaseModel->findAll()
#8 E:\XAMPP\htdocs\system\CodeIgniter.php(932): App\Controllers\Main->homepage()
#9 E:\XAMPP\htdocs\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Main))
#10 E:\XAMPP\htdocs\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 E:\XAMPP\htdocs\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}

Next mysqli_sql_exception: php_network_getaddresses: getaddrinfo for sql108.infinityfree.com failed: No such host is known.  in E:\XAMPP\htdocs\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 E:\XAMPP\htdocs\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('sql108.infinity...', 'if0_34583510', Object(SensitiveParameterValue), 'if0_34583510_ti...', 3306, '', 0)
#1 E:\XAMPP\htdocs\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 E:\XAMPP\htdocs\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 E:\XAMPP\htdocs\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 E:\XAMPP\htdocs\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#5 E:\XAMPP\htdocs\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#6 E:\XAMPP\htdocs\app\Controllers\Main.php(34): CodeIgniter\BaseModel->findAll()
#7 E:\XAMPP\htdocs\system\CodeIgniter.php(932): App\Controllers\Main->homepage()
#8 E:\XAMPP\htdocs\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Main))
#9 E:\XAMPP\htdocs\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 E:\XAMPP\htdocs\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: php_network_getaddresses: getaddrinfo for sql108.infinityfree.com failed: No such host is known.  in E:\XAMPP\htdocs\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 E:\XAMPP\htdocs\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 E:\XAMPP\htdocs\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 E:\XAMPP\htdocs\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 E:\XAMPP\htdocs\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#4 E:\XAMPP\htdocs\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#5 E:\XAMPP\htdocs\app\Controllers\Main.php(34): CodeIgniter\BaseModel->findAll()
#6 E:\XAMPP\htdocs\system\CodeIgniter.php(932): App\Controllers\Main->homepage()
#7 E:\XAMPP\htdocs\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Main))
#8 E:\XAMPP\htdocs\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 E:\XAMPP\htdocs\index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-07-19 05:48:20 --> Unable to connect to the database.
Main connection [MySQLi]: php_network_getaddresses: getaddrinfo for sql108.infinityfree.com failed: No such host is known. 
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `struktur`', [], false)
 3 SYSTEMPATH\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\Main.php(34): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Main->homepage()
 7 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Main))
 8 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-07-19 05:49:53 --> Error connecting to the database: ErrorException: mysqli::real_connect(): php_network_getaddresses: getaddrinfo for sql108.infinityfree.com failed: No such host is known.  in E:\XAMPP\htdocs\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'mysqli::real_co...', 'E:\\XAMPP\\htdocs...', 173)
#1 E:\XAMPP\htdocs\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('sql108.infinity...', 'if0_34583510', Object(SensitiveParameterValue), 'if0_34583510_ti...', 3306, '', 0)
#2 E:\XAMPP\htdocs\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#3 E:\XAMPP\htdocs\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#4 E:\XAMPP\htdocs\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#5 E:\XAMPP\htdocs\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#6 E:\XAMPP\htdocs\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#7 E:\XAMPP\htdocs\app\Controllers\Main.php(34): CodeIgniter\BaseModel->findAll()
#8 E:\XAMPP\htdocs\system\CodeIgniter.php(932): App\Controllers\Main->homepage()
#9 E:\XAMPP\htdocs\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Main))
#10 E:\XAMPP\htdocs\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 E:\XAMPP\htdocs\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}

Next mysqli_sql_exception: php_network_getaddresses: getaddrinfo for sql108.infinityfree.com failed: No such host is known.  in E:\XAMPP\htdocs\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 E:\XAMPP\htdocs\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('sql108.infinity...', 'if0_34583510', Object(SensitiveParameterValue), 'if0_34583510_ti...', 3306, '', 0)
#1 E:\XAMPP\htdocs\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 E:\XAMPP\htdocs\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 E:\XAMPP\htdocs\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 E:\XAMPP\htdocs\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#5 E:\XAMPP\htdocs\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#6 E:\XAMPP\htdocs\app\Controllers\Main.php(34): CodeIgniter\BaseModel->findAll()
#7 E:\XAMPP\htdocs\system\CodeIgniter.php(932): App\Controllers\Main->homepage()
#8 E:\XAMPP\htdocs\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Main))
#9 E:\XAMPP\htdocs\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 E:\XAMPP\htdocs\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: php_network_getaddresses: getaddrinfo for sql108.infinityfree.com failed: No such host is known.  in E:\XAMPP\htdocs\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 E:\XAMPP\htdocs\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 E:\XAMPP\htdocs\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 E:\XAMPP\htdocs\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 E:\XAMPP\htdocs\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#4 E:\XAMPP\htdocs\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#5 E:\XAMPP\htdocs\app\Controllers\Main.php(34): CodeIgniter\BaseModel->findAll()
#6 E:\XAMPP\htdocs\system\CodeIgniter.php(932): App\Controllers\Main->homepage()
#7 E:\XAMPP\htdocs\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Main))
#8 E:\XAMPP\htdocs\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 E:\XAMPP\htdocs\index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-07-19 05:49:53 --> Unable to connect to the database.
Main connection [MySQLi]: php_network_getaddresses: getaddrinfo for sql108.infinityfree.com failed: No such host is known. 
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `struktur`', [], false)
 3 SYSTEMPATH\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\Main.php(34): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Main->homepage()
 7 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Main))
 8 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-07-19 05:49:54 --> Error connecting to the database: ErrorException: mysqli::real_connect(): php_network_getaddresses: getaddrinfo for sql108.infinityfree.com failed: No such host is known.  in E:\XAMPP\htdocs\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'mysqli::real_co...', 'E:\\XAMPP\\htdocs...', 173)
#1 E:\XAMPP\htdocs\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('sql108.infinity...', 'if0_34583510', Object(SensitiveParameterValue), 'if0_34583510_ti...', 3306, '', 0)
#2 E:\XAMPP\htdocs\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#3 E:\XAMPP\htdocs\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#4 E:\XAMPP\htdocs\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#5 E:\XAMPP\htdocs\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#6 E:\XAMPP\htdocs\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#7 E:\XAMPP\htdocs\app\Controllers\Main.php(34): CodeIgniter\BaseModel->findAll()
#8 E:\XAMPP\htdocs\system\CodeIgniter.php(932): App\Controllers\Main->homepage()
#9 E:\XAMPP\htdocs\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Main))
#10 E:\XAMPP\htdocs\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 E:\XAMPP\htdocs\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}

Next mysqli_sql_exception: php_network_getaddresses: getaddrinfo for sql108.infinityfree.com failed: No such host is known.  in E:\XAMPP\htdocs\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 E:\XAMPP\htdocs\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('sql108.infinity...', 'if0_34583510', Object(SensitiveParameterValue), 'if0_34583510_ti...', 3306, '', 0)
#1 E:\XAMPP\htdocs\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 E:\XAMPP\htdocs\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 E:\XAMPP\htdocs\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 E:\XAMPP\htdocs\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#5 E:\XAMPP\htdocs\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#6 E:\XAMPP\htdocs\app\Controllers\Main.php(34): CodeIgniter\BaseModel->findAll()
#7 E:\XAMPP\htdocs\system\CodeIgniter.php(932): App\Controllers\Main->homepage()
#8 E:\XAMPP\htdocs\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Main))
#9 E:\XAMPP\htdocs\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 E:\XAMPP\htdocs\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: php_network_getaddresses: getaddrinfo for sql108.infinityfree.com failed: No such host is known.  in E:\XAMPP\htdocs\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 E:\XAMPP\htdocs\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 E:\XAMPP\htdocs\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 E:\XAMPP\htdocs\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 E:\XAMPP\htdocs\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#4 E:\XAMPP\htdocs\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#5 E:\XAMPP\htdocs\app\Controllers\Main.php(34): CodeIgniter\BaseModel->findAll()
#6 E:\XAMPP\htdocs\system\CodeIgniter.php(932): App\Controllers\Main->homepage()
#7 E:\XAMPP\htdocs\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Main))
#8 E:\XAMPP\htdocs\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 E:\XAMPP\htdocs\index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-07-19 05:49:54 --> Unable to connect to the database.
Main connection [MySQLi]: php_network_getaddresses: getaddrinfo for sql108.infinityfree.com failed: No such host is known. 
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `struktur`', [], false)
 3 SYSTEMPATH\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\Main.php(34): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Main->homepage()
 7 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Main))
 8 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-07-19 05:51:38 --> Array to string conversion
in APPPATH\Views\structure.php on line 200.
 1 APPPATH\Views\structure.php(200): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Array to string conversion', 'E:\\XAMPP\\htdocs\\app\\Views\\structure.php', 200)
 2 SYSTEMPATH\View\View.php(224): include('E:\\XAMPP\\htdocs\\app\\Views\\structure.php')
 3 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1180): CodeIgniter\View\View->render('structure', [], true)
 5 APPPATH\Controllers\Main.php(83): view('structure', [...])
 6 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Main->structure()
 7 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Main))
 8 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-07-19 05:51:39 --> Array to string conversion
in APPPATH\Views\structure.php on line 200.
 1 APPPATH\Views\structure.php(200): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Array to string conversion', 'E:\\XAMPP\\htdocs\\app\\Views\\structure.php', 200)
 2 SYSTEMPATH\View\View.php(224): include('E:\\XAMPP\\htdocs\\app\\Views\\structure.php')
 3 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1180): CodeIgniter\View\View->render('structure', [], true)
 5 APPPATH\Controllers\Main.php(83): view('structure', [...])
 6 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Main->structure()
 7 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Main))
 8 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-07-19 05:51:41 --> Array to string conversion
in APPPATH\Views\structure.php on line 200.
 1 APPPATH\Views\structure.php(200): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Array to string conversion', 'E:\\XAMPP\\htdocs\\app\\Views\\structure.php', 200)
 2 SYSTEMPATH\View\View.php(224): include('E:\\XAMPP\\htdocs\\app\\Views\\structure.php')
 3 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1180): CodeIgniter\View\View->render('structure', [], true)
 5 APPPATH\Controllers\Main.php(83): view('structure', [...])
 6 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Main->structure()
 7 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Main))
 8 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-07-19 05:51:47 --> Array to string conversion
in APPPATH\Views\structure.php on line 200.
 1 APPPATH\Views\structure.php(200): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Array to string conversion', 'E:\\XAMPP\\htdocs\\app\\Views\\structure.php', 200)
 2 SYSTEMPATH\View\View.php(224): include('E:\\XAMPP\\htdocs\\app\\Views\\structure.php')
 3 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1180): CodeIgniter\View\View->render('structure', [], true)
 5 APPPATH\Controllers\Main.php(83): view('structure', [...])
 6 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Main->structure()
 7 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Main))
 8 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-07-19 05:51:57 --> Array to string conversion
in APPPATH\Views\structure.php on line 200.
 1 APPPATH\Views\structure.php(200): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Array to string conversion', 'E:\\XAMPP\\htdocs\\app\\Views\\structure.php', 200)
 2 SYSTEMPATH\View\View.php(224): include('E:\\XAMPP\\htdocs\\app\\Views\\structure.php')
 3 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1180): CodeIgniter\View\View->render('structure', [], true)
 5 APPPATH\Controllers\Main.php(83): view('structure', [...])
 6 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Main->structure()
 7 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Main))
 8 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-07-19 06:02:28 --> Class "App\Models\ArtikelModel" not found
in APPPATH\Controllers\Main.php on line 20.
 1 SYSTEMPATH\CodeIgniter.php(904): App\Controllers\Main->__construct()
 2 SYSTEMPATH\CodeIgniter.php(488): CodeIgniter\CodeIgniter->createController()
 3 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-07-19 06:08:57 --> Undefined variable $artikel
in APPPATH\Views\include\article.php on line 4.
 1 APPPATH\Views\include\article.php(4): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $artikel', 'E:\\XAMPP\\htdocs\\app\\Views\\include\\article.php', 4)
 2 SYSTEMPATH\View\View.php(224): include('E:\\XAMPP\\htdocs\\app\\Views\\include\\article.php')
 3 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\View\View.php(452): CodeIgniter\View\View->render('include/article', null, true)
 5 APPPATH\Views\article.php(3): CodeIgniter\View\View->include('include/article')
 6 SYSTEMPATH\View\View.php(224): include('E:\\XAMPP\\htdocs\\app\\Views\\article.php')
 7 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 8 SYSTEMPATH\Common.php(1180): CodeIgniter\View\View->render('article', [], true)
 9 APPPATH\Controllers\Main.php(116): view('article', [...])
10 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Main->article()
11 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Main))
12 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-07-19 06:31:07 --> Error connecting to the database: ErrorException: mysqli::real_connect(): php_network_getaddresses: getaddrinfo for sql108.infinityfree.com failed: No such host is known.  in E:\XAMPP\htdocs\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'mysqli::real_co...', 'E:\\XAMPP\\htdocs...', 173)
#1 E:\XAMPP\htdocs\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('sql108.infinity...', 'if0_34589920', Object(SensitiveParameterValue), 'if0_34589920_ti...', 3306, '', 0)
#2 E:\XAMPP\htdocs\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#3 E:\XAMPP\htdocs\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#4 E:\XAMPP\htdocs\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#5 E:\XAMPP\htdocs\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#6 E:\XAMPP\htdocs\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#7 E:\XAMPP\htdocs\app\Controllers\Main.php(37): CodeIgniter\BaseModel->findAll()
#8 E:\XAMPP\htdocs\system\CodeIgniter.php(932): App\Controllers\Main->homepage()
#9 E:\XAMPP\htdocs\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Main))
#10 E:\XAMPP\htdocs\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 E:\XAMPP\htdocs\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}

Next mysqli_sql_exception: php_network_getaddresses: getaddrinfo for sql108.infinityfree.com failed: No such host is known.  in E:\XAMPP\htdocs\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 E:\XAMPP\htdocs\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('sql108.infinity...', 'if0_34589920', Object(SensitiveParameterValue), 'if0_34589920_ti...', 3306, '', 0)
#1 E:\XAMPP\htdocs\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 E:\XAMPP\htdocs\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 E:\XAMPP\htdocs\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 E:\XAMPP\htdocs\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#5 E:\XAMPP\htdocs\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#6 E:\XAMPP\htdocs\app\Controllers\Main.php(37): CodeIgniter\BaseModel->findAll()
#7 E:\XAMPP\htdocs\system\CodeIgniter.php(932): App\Controllers\Main->homepage()
#8 E:\XAMPP\htdocs\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Main))
#9 E:\XAMPP\htdocs\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 E:\XAMPP\htdocs\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: php_network_getaddresses: getaddrinfo for sql108.infinityfree.com failed: No such host is known.  in E:\XAMPP\htdocs\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 E:\XAMPP\htdocs\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 E:\XAMPP\htdocs\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 E:\XAMPP\htdocs\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 E:\XAMPP\htdocs\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#4 E:\XAMPP\htdocs\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#5 E:\XAMPP\htdocs\app\Controllers\Main.php(37): CodeIgniter\BaseModel->findAll()
#6 E:\XAMPP\htdocs\system\CodeIgniter.php(932): App\Controllers\Main->homepage()
#7 E:\XAMPP\htdocs\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Main))
#8 E:\XAMPP\htdocs\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 E:\XAMPP\htdocs\index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-07-19 06:31:07 --> Unable to connect to the database.
Main connection [MySQLi]: php_network_getaddresses: getaddrinfo for sql108.infinityfree.com failed: No such host is known. 
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `struktur`', [], false)
 3 SYSTEMPATH\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\Main.php(37): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Main->homepage()
 7 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Main))
 8 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
